crates/nu-cli/src/eval_file.rs

```diff
@@ -107,6 +107,11 @@ pub fn evaluate_file(
     trace!("parsing file: {}", file_path_str);
     let block = parse(&mut working_set, Some(file_path_str), &file, false);
 
+    if let Some(err) = working_set.parse_errors.first() {
+        report_error(&working_set, err);
```

Comment:
Is it meaningful to report all `working_set.parse_errors` if there are some?

Reply:
Looks very verbose :sweat_smile: 

* a.nu
```nu
let a=1
let 0a = 1
let a = '1"
```

./target/debug/nu a.nu

```
Error: nu::parser::unexpected_eof

  × Unexpected end of code.
   ╭─[/data/source/nushell/a.nu:3:1]
 3 │ let a = '1"
   ╰────

Error: nu::parser::parse_mismatch

  × Parse mismatch during operation.
   ╭─[/data/source/nushell/a.nu:1:1]
 1 │ let a=1
   ·     ─┬─
   ·      ╰── expected valid variable name
 2 │ let 0a = 1
   ╰────

Error: nu::parser::missing_positional

  × Missing required positional argument.
   ╭─[/data/source/nushell/a.nu:1:1]
 1 │ let a=1
   ·        ▲
   ·        ╰── missing initial_value
 2 │ let 0a = 1
   ╰────
  help: Usage: let <var_name> = <initial_value>

Error: nu::parser::unexpected_eof

  × Unexpected end of code.
   ╭─[/data/source/nushell/a.nu:3:1]
 3 │ let a = '1"
   ╰────
```

Reply:
OK yeah, that would require some extra work to be helpful. 

Let's get this merged as is for now and we can figure out how to improve that later. (The IDE support already uses those multiple errors to produce squigglies)